extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -333,25 +333,67 @@ describes.realWin('AmpdocAnalyticsRoot', {amp: 1}, (env) => {
       addTestInstance(root3.getElement(body, '#target'), null);
     });
 
-    it('should find an AMP element for AMP search', () => {
+    it('should find an AMP element for AMP search', async () => {
       child.classList.add('i-amphtml-element');
-      return root.getAmpElement(body, '#child').then((element) => {
-        expect(element).to.equal(child);
-      });
+      const element = await root.getAmpElement(body, '#child');
+      expect(element).to.equal(child);
     });
 
-    it('should allow not-found element for AMP search', () => {
-      return root.getAmpElement(body, '#unknown').catch((error) => {
+    it('should allow not-found element for AMP search', async () => {
+      await root.getAmpElement(body, '#unknown').catch((error) => {
         expect(error).to.match(/Element "#unknown" not found/);
       });
     });
 
-    it('should fail if the found element is not AMP for AMP search', () => {
+    it('should fail if the found element is not AMP for AMP search', async () => {
       child.classList.remove('i-amphtml-element');
-      return root.getAmpElement(body, '#child').catch((error) => {
+      await root.getAmpElement(body, '#child').catch((error) => {
         expect(error).to.match(/required to be an AMP element/);
       });
     });
+
+    describe('get amp elements', () => {
```

Comment:
To confirm, what will be the API provided after all changes.
`analyticsRoot.getElements()` and `analyticsRoot.getElement()`
or a single `analyticsRoot.getElements()` or anything?
The design would affect existing tests.

Reply:
We will have `analyticsRoot.getElements()` (added in this pr) and `analyticsRoot.getElement()`, previous tests cover `getElement()`.

Reply:
No, `getElement()` is still used by `getAmpElement()` which is staying (non-experiment case).